### DISCUSSION TITLE
duiLib有什么交流群吗？这个项目目前还在维护吗？

### DISCUSSION THREAD

**Discussion by Polaris-hzn8** [netease-im/NIM_Duilib_Framework#483] [General]
希望拿到duiLib交流群的联系方式，

**Comment by nmgwddj** [comment]
交流群不利于问题的沉淀，使用方法和姿势可通过 github discussions 沟通，有搜索引擎索引方便其他碰到类似问题的人查询。需求和缺陷可通过 github issue 反馈。
